src/api/client.js

```diff
@@ -12,7 +12,20 @@ export function pingStudioHome() {
   );
 }
 
-export function getAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetType = '' }) {
+function assetTypeValueFromState(assetFilters) {
+  // TODO: change this to multi-value once API supports it
+  if (assetFilters.other) {
+    return 'OTHER';
+  } else if (assetFilters.images) {
+    return 'Images';
+  } else if (assetFilters.documents) {
+    return 'Documents';
+  }
```

Comment:
Scratchpad idea around simplifying some of this logic: what if we stored these param names (`OTHER`, `Images`, `Documents`) as the `key`s in `ASSET_TYPES` up above? The reducer state might get a little uglier, since we'd be using stuff like `OTHER` as keys there too, but we wouldn't have to deal with disambiguation around relating arbitrary `key`s to API param names.